package.json

```diff
@@ -10,9 +10,9 @@
         "production": "cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --no-progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js"
     },
     "devDependencies": {
-        "@pipe-dream/core": "^1.0.2",
-        "@pipe-dream/example-file-factory": "^1.0.2",
-        "@pipe-dream/laravel-file-factory": "^1.0.2",
+        "@pipe-dream/core": "^1.0.9",
```

Comment:
Expecting https://github.com/pipe-dream/core/pull/41 to be merged and released first 😄